### ISSUE TITLE
[LOC]Wrong link range (ja-jp)

### ISSUE THREAD

**Issue by hyoshioka0128** [MicrosoftDocs/OfficeDocs-SkypeForBusiness#6982]
[LOC]Wrong link range (ja-jp)

ja-jp:"https://docs.microsoft.com/ja-jp/microsoftteams/shared-line-appearance"
![shared-line-appearance_j](https://user-images.githubusercontent.com/40815708/113697995-f1fa9680-970e-11eb-81e4-0344f3ac5395.png)

en-us:"https://docs.microsoft.com/en-us/microsoftteams/shared-line-appearance"
![shared-line-appearance_e](https://user-images.githubusercontent.com/40815708/113698006-f6bf4a80-970e-11eb-9c9f-d8914d567762.png)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 8d3dd879-564e-62ca-2095-3ef6ae4fab2d
* Version Independent ID: 8ed3dff3-b2b8-b3db-6db9-6bbecdb6cf5c
* Content: [Shared line appearance in Microsoft Teams - Microsoft Teams](https://docs.microsoft.com/en-us/microsoftteams/shared-line-appearance)
* Content Source: [Teams/shared-line-appearance.md](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Teams/shared-line-appearance.md)
* Service: **msteams**
* GitHub Login: @SerdarSoysal
* Microsoft Alias: **serdars**

**Comment by scanum** [comment]
@hyoshioka0128 Thank you for submitting feedback and contributing to the docs. We are currently investigating this.

@officeglobal Could you kindly help with this issue? Thanks

**Comment by TinaMcN** [comment]
@hyoshioka0128 thanks for reporting it.  I sent the article to be human translated and it should be completed in 3 business days

**Comment by TinaMcN** [comment]
@hyoshioka0128 the article has been human translated, can you please close it if it looks ok now?

**Comment by hyoshioka0128** [comment]
@TinaMcN  - One of the two is not fixed. Thanks.
![shared-line-appearance_j2](https://user-images.githubusercontent.com/40815708/114651848-aef87e80-9d1f-11eb-84fb-ae393be06959.png)

**Comment by hyoshioka0128** [comment]
@TinaMcN  - For HTML, the following correction plan. Thanks.
```
「<a href="/ja-jp/office365/servicedescriptions/teams-service-description" data-linktype="absolute-path">Microsoft Teams サービスの説明を参照してください</a>」。
```
 ↓
```
「<a href="/ja-jp/office365/servicedescriptions/teams-service-description" data-linktype="absolute-path">Microsoft Teams サービスの説明</a>」を参照してください。
```

**Comment by TinaMcN** [comment]
@hyoshioka0128 apologies that it wasn't fixed properly.  I have asked the localization vendors to fix it now according to your suggestion.

**Comment by TinaMcN** [comment]
@hyoshioka0128 this has been fixed now.  Please check and close if it's ok.

**Comment by hyoshioka0128** [comment]
@TinaMcN - Fixed. Thanks!
Closed #6982 .
